src/main/java/com/autotune/analyzer/services/UpdateRecommendations.java

```diff
@@ -104,21 +101,26 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
                     LOGGER.debug("UpdateRecommendations API request count: {} success", calCount);
                     interval_end_time = Utils.DateUtils.getTimeStampFrom(KruizeConstants.DateFormats.STANDARD_JSON_DATE_FORMAT,
                             intervalEndTimeStr);
+                    LOGGER.info("Update Recommendation API success response, experiment_name: {} and intervalEndTimeStr : {}",
```

Comment:
Sorry I missed this
Dates should be UTC time zone and format should be 
 = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
 
 
 SimpleDateFormat sdf = new SimpleDateFormat(KruizeConstants.DateFormats.STANDARD_JSON_DATE_FORMAT, Locale.ROOT);
        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
        sdf.format(interval_end_time)

Reply:
Update now